spec/classes/foreman_proxy__config__spec.rb

```diff
@@ -343,11 +343,23 @@
             end
           else
             it { should contain_package('grub').with_ensure('installed') }
-            it 'should copy grub1 files for Red Hat version 6 and older' do
-              should contain_file('/var/lib/tftpboot/grub/grubx64.efi').
-                with_ensure('file').
-                with_owner('root').
-                with_mode('0644')
+            case facts[:operatingsystem]
```

Comment:
This doesn't look correct. The tests passed without this part. It should include the ```should contain_foreman_proxy__tftp__copy_file``` part from above.